go.mod

```diff
@@ -30,18 +30,20 @@ require (
 	golang.org/x/exp v0.0.0-20240604190554-fc45aab8b7f8
 	gopkg.in/k8snetworkplumbingwg/multus-cni.v4 v4.0.2
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.29.5
-	k8s.io/apimachinery v0.29.5
+	k8s.io/api v0.30.2
```

Comment:
We can't switch to v0.30.2

Reply:
sure, I remember you told me. not finished yet with this one, sorry

Reply:
pushed to see all changes in a normal way